src/core/task/index.ts

```diff
@@ -116,6 +116,7 @@ export class Task {
 	isInitialized = false
 	isAwaitingPlanResponse = false
 	didRespondToPlanAskBySwitchingMode = false
+	lastOptionsCount: number = 0
```

Comment:
we dont need this property, instead we can retrieve this from the message data itself (we should avoid adding anymore class properties here than we have to)